snippets/bingads.liquid

```diff
@@ -0,0 +1,6 @@
+

```

Comment:
This brings up an interesting issue. It sure would be nice if we had a "production"/"non-production" flag somwhere. preferably set via C.I. ...  I'm not sure how to do that yet.